client/lib/invites/stores/invites-create-validation.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Internal dependencies
+ */
+import { createReducerStore } from 'lib/store';
+import { reducer, initialState } from 'lib/invites/reducers/invites-create-validation';
+
+const InvitesCreateValidationStore = createReducerStore( reducer, initialState );
+
+InvitesCreateValidationStore.getInvite = ( siteId, usernamesOrEmails ) => InvitesCreateValidationStore.get().getIn( [ 'list', siteId, usernamesOrEmails ] );
+InvitesCreateValidationStore.getInviteError = ( siteId, usernamesOrEmails ) => InvitesCreateValidationStore.get().getIn( [ 'errors', siteId, usernamesOrEmails ] );
```

Comment:
I'm not sure that we'll want `getInvite` and `getInviteError`. I imagine that we'll want to do something like: `getValidationError` and `getValidationSuccesses`. Assuming we leave the API side as is.